gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/domain/SweDichotomyResult.java

```diff
@@ -21,13 +21,19 @@ public class SweDichotomyResult {
     private final DichotomyDirection dichotomyDirection;
     private final DichotomyResult<RaoResponse> dichotomyResult;
     private final Optional<VoltageMonitoringResult> voltageMonitoringResult;
```

Comment:
Sonarlint does not like having an optional as a parameter. Dont change it now but we keep it in mind

Reply:
SonarLint seems to work different on my laptop, no warnings of such on my side or in Github :(